jupyter-notebooks/analysis-ready-data/ard_2_use_case_1.ipynb

```diff
@@ -1,6 +1,7 @@
 {
  "cells": [
   {
+   "attachments": {},
```

Comment:
In the text section below, capitalize the acronym breakdown for NDVI: "Normalized Difference Vegetation Index"

Reply:
Also, define UDM and UDM2 on their first mention, and link to relevant docs